spec/workflow/states/task_spec.rb

```diff
@@ -273,7 +280,11 @@
             .to receive(:run_async!)
             .with(payload["Resource"], input, nil)
 
-          expect { state.run!(input) }.to raise_error(RuntimeError, "Exception")
+          state.run!(input)
+
+          expect(ctx.next_state).to     be_nil
+          expect(ctx.status).to         eq("failure")
+          expect(ctx.state["Error"]).to eq("Exception")
```

Comment:
can we test the `output` here instead of `state["Error"]`?

I think I'm ready to retire those special fields

Reply:
:+1: good call will do